docs/variant_calling.md

```diff
@@ -0,0 +1,53 @@
+# Variant calling
+
+- [Germline variant calling](#germline-variant-calling)
+- [Somatic variant calling with tumor - normal pairs](#somatic-variant-calling-with-tumor---normal-pairs)
+- [Somatic variant calling with tumor only samples](#somatic-variant-calling-with-tumor-only-samples)
+
+## Germline variant calling
+
+Using Sarek, germline variant calling will be performed always, if a variant calling tool is selected and it allows for germline variant calling.
+You can specify the variant caller to use with the `--tools` parameter (see [usage](./usage.md) for more information).
+
+Germline variant calling can currently only be performed with the following variant callers:
+
+- Haplotypecaller
+- Manta
+- Sentieon (check the specific [sentieon](TODO add link) documentation)
+- Strelka
+- Tiddit
+
+For more information on the individual variant callers, and where to find the variant calling results, check the [output](output.md) documentation.
+
+## Somatic variant calling with tumor - normal pairs
+
+Using Sarek, somatic variant calling will be performed, if your input tsv file contains tumor / normal pairs (see [input](input.md) documentation for more information).
+Different samples belonging to the same patient, where at least one is marked as normal (`0` in the `Status` column) and at least one is marked as tumor (`1` in the `Status` column) are treated as tumor / normal pairs.
+
+If tumor-normal pairs are provided, both germline variant calling and somatic variant calling will be performed, provided that the selected variant caller allows for it.
+If the selected variant caller allows only for somatic variant calling, then only somatic variant calling results will be generated.
+
+Here is a list of the variant calling tools that support somatic variant calling:
+
+- Ascat (check the specific [Ascat](ascat.md) documentation)
+- ControlFREEC
+- Freebayes
+- Manta
+- Mpileup
+- MSIsensor
+- Mutect2
+- Sentieon (check the specific [sentieon](TODO add link) documentation)
+- Strelka
+
+For more information on the individual variant callers, and where to find the variant calling results, check the [output](output.md) documentation.
+
+## Somatic variant calling with tumor only samples
+
+Somatic variant calling with only tumor samples (no matching normal sample), is not recommended by the GATK best practices.
+This is just supported for a limited variant callers.
+
+Here is a list of the variant calling tools that support tumor-only somatic variant calling:
+
+- Mutect2
```

Comment:
I think `Manta` does have a tumor only mode as well.
And we do run `TIDDIT` and `HaplotypeCaller` on the tumor as well, just to have more informations